k8s/apis/workflow/v1/activation_types.go

```diff
@@ -17,10 +17,9 @@ import (
 type ActivationStatus struct {
 	Status               v1alpha2.State `json:"status,omitempty"`
 	StatusMessage        string         `json:"statusMessage,omitempty"`
-	ErrorMessage         string         `json:"errorMessage,omitempty"`
```

Comment:
We don't need the error message anymore?

Reply:
It's recorded per stage